types/ember__object/test/computed.ts

```diff
@@ -179,32 +179,53 @@ const objectWithComputedProperties = EmberObject.extend({
     setDiff: setDiff('foo', 'bar'),
     sort1: sort('foo', 'bar'),
     sort2: sort('foo', (itemA, itemB) => {
-        if (itemA < itemB) {
-            return -1;
-        } else if (itemA > itemB) {
-            return 1;
-        } else {
-            return 0;
-        }
+      return `${itemA}`.length - `${itemB}`.length;
     }),
     sort3: sort('foo', ['bar', 'baz'], (itemA, itemB) => {
-        if (itemA < itemB) {
-            return -1;
-        } else if (itemA > itemB) {
-            return 1;
-        } else {
-            return 0;
-        }
+      return `${itemA}`.length - `${itemB}`.length;
     }),
 
     sum: sum('foo'),
     union: union('foo', 'bar', 'baz', 'qux'),
     uniq: uniq('foo'),
     uniqBy: uniqBy('foo', 'bar'),
-});
-
-const component2 = EmberObject.extend({
-    isAnimal: or('isDog', 'isCat'),
 }).create();
 
-assertType<boolean>(component2.get('isAnimal'));
+assertType<unknown>(objectWithComputedProperties.get('alias'));
```

Comment:
Is there a good way to actually assert that something has a type of `unknown`? This ends up being worthless, since `assertType<unknown>(anything);` will always work.

Reply:
```ts
objectWithComputedProperties.get('alias'); // $ExpectType unknown
````
(known unkown is OK)

Reply:
@peterblazejewicz's suggestion is the right one—you can verify for yourself that if you get something more specific back than that, dtslint will error:

```ts
const s = "hello"; // $ExpectType unknown
```

will produce

```
Error: /Users/chris/dev/DefinitelyTyped/DefinitelyTyped/types/ember__object/test/computed.ts:194:1
ERROR: 194:1  expect        TypeScript@4.6 expected type to be:
  unknown
got:
  "hello"
```

Reply:
Ah, good. I was just basing this on the type compilation initially working, since I've been having trouble actually running the tests. Thanks!